### PR TITLE
Applicant form #48

### PR DESCRIPTION
**Pull request by amykarnaze**
#### Type of Change Made 
- [ ] bugfix 
- [X] new feature 
- [ ] breaking change 
- [ ] testing 
- [ ] formatting 
- [X] styling

#### What's this PR do?
This PR begins the process of creating and posting a new applicants profile. 

- Fetches all attributes (skills and values) in the 'useEffect' and the response is set to state. Attributes are mapped and displayed on the page and on the refresh icon. The attributes are turned into buttons so they can be clicked and contain name and id. An onClick was started that calls tagHandler. Next steps will be to hook this up so the chosen attribute buttons id can be accessed for a post.

- 'useReducer' manages the state transitions of the form based on when a user types in the inputs. 
handleFormChange dispatches the form inputs. Username is added to ' useState' as it is a disabled input. The username is dispatched in 'useEffect'.

- Styling is added and the icon refreshes/changes upon page load.

#### Where should the reviewer start?
ApplicantForm.tsx
ApplicantForm.scss

#### How should this be manually tested?
Go to 'http://localhost:3000/create-applicant' and play around!

#### What are the relevant tickets?
#48 

#### Screenshots (if appropriate)
Screen Shot 2020-10-29 at 10 46 47 PM

#### Questions:
Should we keep SyntheticEvent for typescript?

### CODE REVIEW

File: src/components/ApplicantForm/ApplicantForm.tsx [hunk 1]
```diff
@@ -1,54 +1,144 @@
...
 				</div>
-				<h5>Select your skills and values</h5>
 				<div className="applicant-attributes">
-					<div className="attribute-box">
+				<h5>Select your skills and values</h5>
 						<h5>Skills</h5>
-						skills were here
-					</div>
 					<div className="attribute-box">
+						{makeTags(tags.skills)}
```
Comment (ErinUntermeyer) [inline src/components/ApplicantForm/ApplicantForm.tsx:133]: Yay for this working! I can't wait until we have a ton to choose from.

**Review by ErinUntermeyer — APPROVED** [review]
Thanks for driving and great work!

File: src/components/ApplicantForm/ApplicantForm.tsx [hunk 2]
```diff
@@ -1,54 +1,144 @@
-import React, { useState } from 'react'
+import { type } from 'os'
+import React, { useState, useEffect, useReducer, SyntheticEvent } from 'react'
 
-import { getNames } from '../../assets/api-calls'
+import { getNames, getSearchOptions } from '../../assets/api-calls'
 import './ApplicantForm.scss'
 
+interface Creator {
```
Comment (GreysonElkins) [inline src/components/ApplicantForm/ApplicantForm.tsx:7]: This wasn't used after all, huh?

**Review by GreysonElkins — APPROVED** [review]
Great job Amy!

**Pull request merged**